Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim
+
+WORKDIR /root/
+
+COPY requirements.txt .
+
+RUN mkdir output &&\
```

Comment:
Split this into separate run statements that Docker can properly cache the layers

Reply:
What would be the benefit of caching these few installation layers separately?

Reply:
Doesn't docker still cache this RUN command?